doc/user/content/ingest-data/cdc-sql-server.md

```diff
@@ -171,23 +171,15 @@ cluster, use the `IN CLUSTER` clause.
 #### Transaction support
 
 Debezium provides [transaction metadata](https://debezium.io/documentation/reference/connectors/sqlserver.html#sqlserver-transaction-metadata)
-that can be used to preserve transactional boundaries downstream. Work is in
-progress to utilize this topic to support transaction-aware processing in
-[Materialize #7537](https://github.com/MaterializeInc/materialize/issues/7537)!
```

Comment:
This didn't actually land, and we have no plans to actually support it.

Reply:
Added patch to add txn metadata to the limitations section
https://github.com/MaterializeInc/materialize/pull/29150/commits/765bc47ee6ab3be240babda9f8c1c85323193ce6